### PR TITLE
add averageOfNumbers in JavaScript

### PR DESCRIPTION
**Pull request by ellu-laitinen**
I added a Javascript function that calculates the average of numbers

### CODE REVIEW

File: src/javascript/ellulaitinen-averageOfNumbers.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// This function calculates the average of numbers
+
+const numbers = [5, 8, -7, 3, -2, 15, 8, 6];
+
+const avgOfNumbers = () => {
```
Comment (jazibjafri) [inline src/javascript/ellulaitinen-averageOfNumbers.js]: @ellu-laitinen make the function name as same as in the filename, or vice versa.

**Review by jazibjafri — CHANGES_REQUESTED** [review]

File: src/javascript/ellulaitinen-averageOfNumbers.js (hunk 1 above)
Comment (ellu-laitinen) [reply src/javascript/ellulaitinen-averageOfNumbers.js]: Oops, sorry. Changed it!

**Comment by jazibjafri** [comment]
@ellu-laitinen your username also does not match

**Comment by ellu-laitinen** [comment]
File name changed now.

**Comment by jazibjafri** [comment]
> File name changed now.

@ellu-laitinen you added another file.

**Review by jazibjafri — APPROVED** [review]

**Pull request merged**